_updates/2021-07-14.html

```diff
@@ -2,17 +2,38 @@
 ---
 <div class="ds-u-padding-y--2">
   <div class="ds-u-font-size--h3 ds-u-font-weight--bold">
-    Version 2 of BCDA is now live, conforming with the FHIR R4 specification!
+    Version 1 of the BCDA Partially Adjudicated Claims Enhancement live for use on May 16!
```

Comment:
I didn't see this change on the Home Page Draft; is there another document that this is referenced in? 🤔

Reply:
As an additional note/disclaimer: I think I might be a little fuzzy on these update pages --  how do these updates work?